### ISSUE TITLE
Disable Capture hotkey

### ISSUE THREAD

**Issue by Creamers158** [Ginj-capture/Ginj#20]
There is not option (at least I don't know) to disable the hotkey function so there is no key registered anymore.

Also when setting a bracket as hotkey I noticed this:

![couldnotregister](https://user-images.githubusercontent.com/6506666/88901163-bdb20a80-d250-11ea-8c4f-3a3633d49b7c.png)

**Comment by vicnevicne** [comment]
You're right about "undefining" the capture combination. Probably I should do it like Jing did: A field that can only be modified when it gets the focus and a "Clear hotkey" button. By the way, I just noticed Jing did forbit many keystrokes. It's proably wise to prevent things like single letter (just tried with P and it becomes impossible to type text ;-)). Probably either Alt or Ctrl should be mandatory in the combination.
As to why the bracket is refused, no idea. Just to be clear, do you mean parenthesis ')', square bracket ']' or curly bracket '}' ? I cannot reproduce the above message on my international keyboard... And did you try combining it with alt or other modifiers ? 
Anyway I'm afraid I'm dependent on the JKeyMaster lib regarding what's rejected. However I'm wondering if I shouldn't switch to https://github.com/kwhat/jnativehook and see if it's better. The fact that it doesn't recognize the PrintScreen key (issue #18 ) is annoying for a screen capture software...

**Comment by vicnevicne** [comment]
The layout change is done, with a "Clear" button. It's much more intuitive (thanks) and it works :-) (silly bug)

But still I don't know how to make it accept the bracket.
I checked JKeyMaster code and all it does is a Windows call to register the key, which fails;
`        if (User32.INSTANCE.RegisterHotKey((HWND)null, id, KeyMap.getModifiers(hotKey.keyStroke), code)) ...`

Note that, thinking more, it is very different from the PrintScreen issue: 
- PrintScreen is not picked up as an event by the java TextField, but I just tested and it can be mapped by hand via the configuration file, even if it's not very intuituve (I'll add that info to the issue).
- the bracket however is perfectly detected but the error occurs when assigning it to the capture action. 

Most probably something at Windows level prevents such an assignment. Did you try using that closing bracket in another app with global hotkey support ?
Unfortunately I cannot replicate that issue because square bracket must be typed using a combination on my layout...
Can you try other combinations, such as opening bracket, or Shift+closing bracket, etc ?
Thanks.

**Comment by vicnevicne** [comment]
In v0.3.9, just released:
- clearing up hotkey should work
- PRINTSCREEN can be used by editing settings file
- closing bracket handling has not changed so I guess it still doesn't work. If you could perform the tests above, maybe it could help nail down the issue...

**Comment by Creamers158** [comment]
Tried shift+closing bracket, alt+closing brack , ctrl+alt+closing bracket, ctrl+open bracket.  Even Backslash, but doesn't work.'
Dead acute doesnt work either. Strangly the Slash does work.

**Comment by vicnevicne** [comment]
OK. Thanks.
I admit I'm out of options for the bracket key (Can you confirm all the cases above cause a "Could not register hotkey" message ?). 
I'll leave the issue open but don't expect a solution for those keys, partly because all those keys (open/close brackets, dead acute and backslash) require a key combination on my keyboard so I cannot reproduce the issue, but also because Java handling of "special keys" seems to be clumsy. See for example [this issue of TurboVNC](https://github.com/TurboVNC/turbovnc/issues/194).

**Comment by Creamers158** [comment]
I have a dualboot option and tested it there )dont have another pc on holiday: ) to see if it has something to do with my windows installation. On this second installation I have the same results
![2020-08-05_18-37-54](https://user-images.githubusercontent.com/6506666/89440220-9c1eba80-d74b-11ea-8e71-b9778ab3d6c1.png)

Another 'potential' problem I found while testing cause it didnt remember my last settings.  I couldnt find the settings.properties file, it appears that it's only created on the first capture and not at runtime.
